pip-requirements.txt

```diff
@@ -0,0 +1,2 @@
+mock>=2.0.0
+coveralls
```

Comment:
@EltayebAhmed. Please make github issues for your repository about the improvements that are not going to make this PR. @martyngigg has highlighted a few that are important but are not required for this PR.